compiler/src/iree/compiler/Dialect/HAL/Transforms/Passes.cpp

```diff
@@ -83,6 +83,27 @@ static llvm::cl::opt<std::string> clSubstituteExecutableSourcesFrom{
     llvm::cl::init(""),
 };
 
+static llvm::cl::list<std::string> clSubstituteExecutableConfiguration{
+    "iree-hal-substitute-executable-configuration",
+    llvm::cl::desc(
+        "A `executable_name=object_file.xxx` pair specifying a "
```

Comment:
This description is copied over from `iree-hal-substitute-executable-object` and needs to be updated?